### PR TITLE
Updated documentation of inequalities

### LINKED ISSUE #23880: Make inequalities solver page easier to understand
The [inequalities solver page](https://docs.sympy.org/dev/modules/solvers/inequalities.html) is currently confusing to visitors because there is no indication of which solver to use for the general case. Suggest adding a paragraph at the top of the page (source at `doc/src/modules/solvers/inequalities.rst`) explaining that `reduce_inequalities` is the top-level function which will call the others as needed.

Visitors are likely to find this page via a web search, so putting a note on it would help them.

### PR DESCRIPTION
**Pull request by metabiswadeep**
#### Release Notes

- other
  - an addition to documentation about inequalities solver.

### CODE REVIEW

File: doc/src/modules/solvers/inequalities.rst [hunk 1]
```diff
@@ -3,6 +3,9 @@
 Inequality Solvers
 ==================
 
+For general cases the reduce_equalities function should be used.
```
Comment (bertiewooster) [inline doc/src/modules/solvers/inequalities.rst:6]:
```suggestion
For general cases :func:`~.reduce_equalities` should be used.
```

Just adding a link to the cited function, so visitors can jump directly to it.

File: doc/src/modules/solvers/inequalities.rst [hunk 2]
```diff
@@ -3,6 +3,9 @@
...
+Other functions are the subcategories useful for special dedicated operations.
```
Comment (bertiewooster) [inline doc/src/modules/solvers/inequalities.rst:7]:
```suggestion
Other functions are the subcategories useful for special dedicated operations, and will be called internally as needed by ``reduce_equalities``.
```

**Review by bertiewooster — CHANGES_REQUESTED** [review]
Thanks for doing this!

**Comment by metabiswadeep** [comment]
Thanks for the suggestions @bertiewooster .

**Comment by asmeurer** [comment]
@bertiewooster do you think this is ready to be merged?

**Comment by bertiewooster** [comment]
@asmeurer yes

**Pull request merged**